src/amqp.c

```diff
@@ -256,6 +264,14 @@ int8_t amqp_publish_persistent_message(const char *exchange,
 	return rc;
 }
 
+/**
+ * amqp_declare_new_queue:
+ * @name: queue name
+ *
+ * This function declare a durable queue in amqp.
```

Comment:
Remove "This", can be: "Declare a durable queue in amqp"

Reply:
Done